tests/acceptance/features/apiSharingNg1/listPermissions.feature

```diff
@@ -2296,4 +2296,99 @@ Feature: List a sharing permissions
           }
         }
       }
+      """
+
+
+  Scenario: user lists permissions of a folder in personal space after enabling 'Space Editor Without Versions' role
```

Comment:
this tole is for sharing space not a folder. hmm, should the role be listed for folder as well?

CC @ScharfViktor

Reply:
Ahh, scenario title is misleading.
```suggestion
  Scenario: user lists permissions of a space after enabling 'Space Editor Without Versions' role
```